test/algorithms/simulation/sidb/exhaustive_ground_state_simulation.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include <catch2/catch_template_test_macros.hpp>
 
+#include "catch2/matchers/catch_matchers_floating_point.hpp"
```

Comment:
Sorry to be nitpicky, but could you use `<>` for library includes? Applies to both files.